lib/utils.js

```diff
@@ -29,18 +29,53 @@ const path = require('path');
 const electron = require('electron');
 const app = electron.app || electron.remote.app;
 
+// define the defaultDataPath and current directory
+const defaultBasePath = 'storage';
+let currentDirectory = '';
+
+/**
+ * @summary Parse directory to avoid parent directories
+ * @function
+ * @private
+ *
+ * @returns {Strings} parsed user data path without parent directories
+ */
+function getDirectory(directory){
+  const directoryArray = directory.split(path.posix.sep);
+  let newDirectory = "";
+  for(var dir of directoryArray){
+    if(dir !== ".."){
+      newDirectory += dir + "/";
+    }
+  }
+  return newDirectory;
+}
+
+/**
+ * @summary Set default user data directory path
+ * @function
+ * @public
+ *
+ * @example
+ * utils.getUserDataPath('newDirectory');
+ */
+exports.setUserDataPath = function(directory) {
```

Comment:
Based on https://github.com/jviotti/electron-json-storage/pull/51/files#r105062802, I think we should do a sanity check here to ensure that `directory` is an absolute path, using `path.isAbsolute()`, and throw an error otherwise.